src/app/commands/first-quest/FirstQuest.ts

```diff
@@ -1,25 +1,31 @@
 import {
-	ApplicationCommandPermissionType,
 	CommandContext,
 	CommandOptionType,
 	SlashCommand,
 	SlashCreator,
 } from 'slash-create';
-import roleIds from '../../service/constants/roleIds';
 import ServiceUtils from '../../utils/ServiceUtils';
 import ConfigureFirstQuest from '../../service/first-quest/ConfigureFirstQuest';
 import ValidationError from '../../errors/ValidationError';
 import discordServerIds from '../../service/constants/discordServerIds';
 import { LogUtils } from '../../utils/Log';
 import FirstQuestPOAP from '../../service/first-quest/FirstQuestPOAP';
+import { sendFqMessage, switchRoles } from '../../service/first-quest/LaunchFirstQuest';
+import fqConstants from '../../service/constants/firstQuest';
```

Comment:
should be import firstQuest from '../../service/constants/firstQuest';

Are there mmore instances of this in other files?